Pet-Clinic-Frontend/src/routes.tsx

```diff
@@ -0,0 +1,6 @@
+export const routes = [
```

Comment:
I think we should rely on some typing here to avoid human mistakes.

Reply:
I don't understand what is wrong? the word  " routes "?

Reply:
No, not really. Nothing is preventing you to have a typo or duplicated fields and so on. If we are using typescript, we might wanted to use the types too.